src/index.js

```diff
@@ -64,7 +64,7 @@ class QueryBuilder extends BaseBuilder {
 	* @return An elasticsearch query
```

Comment:
Added some comments with a link to the elasticsearch documentation page that gives the full explanation.